tools/Customization/DevHome.Customization/Views/DeveloperFileExplorerPage.xaml

```diff
@@ -0,0 +1,29 @@
+<Page
+    x:Class="DevHome.Customization.Views.DeveloperFileExplorerPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:behaviors="using:DevHome.Common.Behaviors"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:views="using:DevHome.Customization.Views"
+    behaviors:NavigationViewHeaderBehavior.HeaderMode="Never"
+    mc:Ignorable="d">
+
+    <Grid MaxWidth="{ThemeResource MaxPageContentWidth}" Margin="{ThemeResource ContentPageMargin}">
```

Comment:
Why are you setting `MaxWidth`

Reply:
This seems to be standard practice with ToolPages, just following preexisting patterns.